pipeline/dags/mrms_download.py

```diff
@@ -0,0 +1,39 @@
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+
+from datetime import timedelta
+
+# default arguments for each task
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    'catchup': False
+}
+
+with DAG(
+    'weather_MRMS_data',
+    default_args=default_args,
+    description='Download MRMS data',
+    schedule_interval=timedelta(days=1),
+    start_date=days_ago(2),
+    tags=['example'],
+) as dag:
+
+    t1 = BashOperator(
+        task_id='print_date',
+        bash_command='date'
+    )
+
+    t2 = BashOperator(
+        task_id='sleep',
+        depends_on_past=False,
+        bash_command='sleep 5',
+        retries=3,
+    )
```

Comment:
Ah.... I finally get it!

This is basically just one of the Airflow examples that hasn't been fully converted to run the actual download scripts.

In other words, a work-in-progress kind of thing.

Got it (sorry to be dense).